app/view_models/bids_new_view_model.rb

```diff
@@ -0,0 +1,167 @@
+class BidsNewViewModel
```

Comment:
how are we feeling about this naming convention? I kind of hate it. I originally moved in this direction because I wanted to know what was a presenter and what was a view model without look at where they were..but in this new world of view models everywhere maybe concise names would be better.  eg: `BidsNew` or `NewBid` instead of `BidsNewViewModel` -- thoughts  @adelevie @harrisj ?